deno/src/context.ts

```diff
@@ -1,5 +1,6 @@
 // deno-lint-ignore-file camelcase
-import { Api, Other } from './core/api.ts'
+import { Api, Other as OtherApi } from './core/api.ts'
```

Comment:
Maybe Other should get a better name. Something more expressive. `OtherArguments` maybe?